spec/features/admin/page_destroy_feature_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'spec_helper'
+
+module Alchemy
+  describe "Page destroy feature", js: true do
+    before { authorize_user(:as_admin) }
+
+    context 'destroying a content page' do
+      let!(:content_page) { create(:alchemy_page) }
+
+      it "redirects to page tree" do
```

Comment:
Good catch. I like to test the actual removing of the page from the tree. Thanks!